test/sequential/test-tls-honorcipherorder.js

```diff
@@ -24,7 +24,9 @@ function test(honorCipherOrder, clientCipher, expectedCipher, cb) {
     secureProtocol: SSL_Method,
     key: fs.readFileSync(common.fixturesDir + '/keys/agent2-key.pem'),
     cert: fs.readFileSync(common.fixturesDir + '/keys/agent2-cert.pem'),
-    ciphers: 'DES-CBC-SHA:AES256-SHA:RC4-SHA:ECDHE-RSA-AES256-SHA',
+    ciphers: common.hasFipsCrypto? // No 56-bit DES or RC4 in FIPS
+        'DES-CBC3-SHA:AES256-SHA:ECDHE-RSA-AES256-SHA' :
+        'DES-CBC-SHA:AES256-SHA:RC4-SHA:ECDHE-RSA-AES256-SHA',
```

Comment:
I made this test by using TLS1.0 ciphers in several years ago but I think it can be changed TLS1.2 base so that it is free from FIPS restriction. Let me check this tomorrow.